### ISSUE TITLE
Rename record components of output to match GAUSS pkg

### ISSUE THREAD

**Issue by ssiccha** [jnkbrachter/GaussPar#10]
Atm we can check for correctness by doing e.g.
```
correct := -1 * resStd.vectors = resPar.remnant
           and -1 * resStd.coeffs = resPar.transformation;
```
where `resStd` is the result of `EchelonMatTransformation` and `resPar` is the result of `ChiefParallel`.

**Comment by ssiccha** [comment]
@emmakatherina

**Comment by emmakatherina** [comment]
I am working on that.

**Comment by ssiccha** [comment]
@emmakatherina The function `EchelonMatTransformation` from the [Gauss pkg](https://www.gap-system.org/Packages/gauss.html) does the same as `ChiefParallel`. The result is a record in both cases, but the names of the record components are different.

They should be renamed to match the names of the Gauss pkg.

**Comment by emmakatherina** [comment]
Issue is fixed with PR #20. Please close.
